content/docs/for-buildpack-authors/how-to/write-buildpacks/create-slice-layers.md

```diff
@@ -3,8 +3,34 @@ title="Create slice layers"
 weight=99
 +++
 
+A `Slice` represents a layer in the `<app>` directory to be exported during the `export` phase.
```

Comment:
```suggestion
The contents of the application directory (after all buildpacks have executed) will be included in the final application image as a single layer, OR as `slice` layers. A `slice` layer contains a portion of the application directory as defined by a filepath glob.

Slices are useful to avoid re-uploading unchanged data to the image registry. For example, if the application directory is a monolithic repository containing code for both a web frontend and sever backend, buildpacks can slice the directory into separate layers, so that developers iterating on one part of the code base don't have to wait for the entire directory to re-upload at the end of the build.
```

Reply:
updated